### PR TITLE
Small Heretics Buff

### PR DESCRIPTION
**Pull request by eeSPee**
## About The Pull Request

Heretic' Living Heart no longer has a range limit.
Mansus grasp now always silences for 3 seconds.
Made the recipes for curses a bit easier to obtain.

EDIT: Adds

Rust blade now deals tox damage, the lower the heretic's health, the more damage it deals.
Rust Ascension is now spaceproof!

## Why It's Good For The Game

General changes that make heretics more on par with other antags.

## Changelog
:cl:
tweak: Heretic' Living Heart no longer has a range limit
add: Mansus grasp now always silences for 3 seconds
balance: Made the recipes for curses a bit easier
add: Rust blade now deals tox damage, the lower the heretic's health, the more damage it deals.
balance: Rust ascension now makes you spaceproof
/:cl:

### CODE REVIEW

**Comment by heepox** [comment]
nice. but Z levels screw with this dont they? ah no im thinking of GLOB.poi_list targeting nvm.

**Review by PowerfulBacon — DISMISSED** [review]
The descriptions need to be updated too

**Comment by NickEmbased** [comment]
Awesome

File: code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm [hunk 1]
```diff
@@ -136,7 +136,7 @@
 	gain_text = "Corrupt their flesh, make them bleed."
 	desc = "Curse someone for 5 minutes of inability to walk. Using a knife, pool of blood, left leg, right leg, a hatchet and an item that the victim touched  with their bare hands. "
```
Comment (heepox) [inline code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm]: Curse someone for 5 minutes with the inability to walk. Using a left leg, a right leg, a hatchet and an item that the victim touched with their bare hands. 

should be the new desc for this

File: code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm [hunk 2]
```diff
@@ -119,7 +119,7 @@
 	gain_text = "Cursed land, cursed man, cursed mind."
 	desc = "Curse someone for 2 minutes of vomiting and major organ damage. Using a wirecutter, a spill of blood, a heart, left arm and a right arm, and an item that the victim touched  with their bare hands."
```
Comment (heepox) [inline code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm]: Curse someone for 2 minutes of vomiting and major organ damage. Using wirecutters, a heart and an item that the victim touched with their bare hands.

File: code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm [hunk 3]
```diff
@@ -76,7 +76,7 @@
 	gain_text = "Blind man walks through the world, unnoticed by the masses."
 	desc = "Curse someone with 2 minutes of complete blindness by sacrificing a pair of eyes, a screwdriver and a pool of blood, with an object that the victim has touched with their bare hands."
```
Comment (heepox) [inline code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm]: Curse someone with 2 minutes of complete blindness by sacrificing a pair of eyes and a screwdriver with an object that the victim has touched with their bare hands.

**Comment by eeSPee** [comment]
Thanks for the feedback! The descs should be up to date/explicit enough.

File: code/modules/antagonists/eldritch_cult/knowledge/rust_lore.dm [hunk 4]
```diff
@@ -118,16 +119,19 @@
 
 /datum/eldritch_knowledge/final/rust_final
 	name = "Rustbringer's Oath"
-	desc = "Bring 3 corpses onto the transmutation rune. After you finish the ritual rust will now automatically spread from the rune. Your healing on rust is also tripled, while you become more resillient overall."
+	desc = "Bring 3 corpses onto the transmutation rune. After you finish the ritual rust will now automatically spread from the rune. Your healing on rust is also tripled, while you become more resillient overall and spaceproof."
```
Comment (utilbird) [inline code/modules/antagonists/eldritch_cult/knowledge/rust_lore.dm]: space proof is two words

File: code/modules/antagonists/eldritch_cult/knowledge/rust_lore.dm [hunk 5]
```diff
@@ -77,17 +77,18 @@
 /datum/eldritch_knowledge/rust_blade_upgrade
 	name = "Toxic blade"
 	gain_text = "Let the blade guide you through the flesh."
-	desc = "Your blade of choice will now add toxin to enemies bloodstream."
+	desc = "Your blade of choice will now transfer your pain as toxic damage."
```
Comment (utilbird) [inline code/modules/antagonists/eldritch_cult/knowledge/rust_lore.dm:80]: Do admins do extra damage?

**Review by utilbird — APPROVED** [review]

**Pull request merged**